tools/verdict-back-ends/verdict-bundle-parent/verdict-attack-defense-collector/src/main/java/com/ge/verdict/attackdefensecollector/Triple.java

```diff
@@ -0,0 +1,39 @@
+package com.ge.verdict.attackdefensecollector;
+
+import java.util.Objects;
+
+/**
+ * A generic pair. Correctly implements equals() and hashCode(), so may be used as a key to a hash
+ * map. Note that the L and R types must themselves correctly implement equals() and hashCode() for
+ * this to work correctly.
+ *
```

Comment:
Javadoc comes from Pair and should be updated for this Triple class.

Reply:
fixed in a7b9701